openwisp_notifications/templates/emails/batch_email.txt

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+[{{ site_name }}] {{ notifications_count }} {% translate "new notifications since" %} {{ start_time }}
+
+{% for notification in notifications %}
+- {{ notification.message }}
+  {% translate "URL" %}: {{ notification.url }}
+  {% translate "Timestamp" %}: {{ notification.timestamp|date:"F j, Y, g:i a" }}
```

Comment:
Timestamp should be moved after description and should be named "Date & Time".

Reply:
Updated.